test/alternator/test_cql_rbac.py

```diff
@@ -669,6 +672,167 @@ def test_rbac_updatetable(dynamodb, cql):
                     authorized(lambda: tab.meta.client.update_table(TableName=tab.name,
                         BillingMode='PAY_PER_REQUEST'))
 
+# Above we checked UpdateTable's operation to change BillingMode, and that
```

Comment:
"Above" - that is, unless sombody doesn't insert a testcase in between.

Reply:
Yes, I'm writing a book here :-)
Also above here doesn't mean the immediately above test, just that we had another test for it. When I want to refer to a specific test I usually include its name exactly for the reason you asked about. I can do it here too.

Reply:
Fixing the text to "Above in test_rbac_updatetable() we checked". 

I know nothing guarantees that it will remain "above", but I think there's a high likelyhood that nobody will ever come to this test file and shuffle around functions randomly, so the word "above" is likely to remain correct (and if not, it can be fixed when becomes incorrected).

Reply:
I'm trying to do this but lacking the git skills, apparently :-( I'm still thinking how to do it without losing my mind.

Reply:
I managed to do it :-) Mostly using "git rebase -i" as the tool.